packages/garbo/src/embezzler/fights.ts

```diff
@@ -223,7 +223,8 @@ export const chainStarters = [
     "Scepter Semirare",
     () =>
       canAdventure($location`Cobb's Knob Treasury`) &&
-      shouldAugustCast($skill`Aug. 2nd: Find an Eleven-Leaf Clover Day`),
+      shouldAugustCast($skill`Aug. 2nd: Find an Eleven-Leaf Clover Day`) &&
+      globalOptions.target === $monster`Knob Goblin Embezzler`,
```

Comment:
depending on how often we are doing this, it may be better to make something like `globalOptions.target === embezzler` or even `targetEmbezzlers()`. seems fine as is tho

Reply:
Gonna merge as-is but probably do a `targetEmbezzlers` once we start doing things like removing robort prep and meat buffing